tracks/kotlin/exercises/rotational-cipher/mentoring.md

```diff
@@ -0,0 +1,42 @@
+### Reasonable Solutions
+
+```kotlin
+class RotationalCipher(private val key: Int) {
+    init {
+        require(key in 0..26) {
+            "Key should be integer between 0 and 26"
+        }
+    }
+
+    fun encode(input: String): String =
+            input.map(shiftBy(key)).joinToString("")
+
+    // Do not allow private members to access object state.
```

Comment:
I don't think this commentary makes sense here. Sure, it's good advice, but doesn't add to this solution. Instead add a talking point 💯